gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/analytics/ApiAnalyticsResource.java

```diff
@@ -115,8 +115,10 @@ public ApiAnalyticsAverageConnectionDurationResponse getAverageConnectionDuratio
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @Permissions({ @Permission(value = RolePermission.API_ANALYTICS, acls = { RolePermissionAction.READ }) })
-    public ApiAnalyticsResponseStatusRangesResponse getResponseStatusRanges() {
-        var request = new SearchResponseStatusRangesUseCase.Input(apiId, GraviteeContext.getCurrentEnvironment());
+    public ApiAnalyticsResponseStatusRangesResponse getResponseStatusRanges(@QueryParam("from") Long from, @QueryParam("to") Long to) {
+        long end = to != null ? to : Instant.now().toEpochMilli();
```

Comment:
This (and next line) should be in the usecase. It is part of the business rules and there should be tested at the use case level